install/runtime-payload/scripts/kata-deploy.sh

```diff
@@ -250,6 +303,9 @@ function main() {
 			[ -x $(command -v containerd) ]; then
 			containerd config default > "$containerd_conf_file"
 		fi
+	# CRI-O isn't consistent with the naming -- let's use crio to match the service file
+	elif [ "$runtime" == "cri-o" ]; then
+		runtime="crio"
```

Comment:
Maybe this part of the code should belong to the "get_container_runtime" function?
This would make the function reliably return the runtime's name, wherever it is called

Reply:
I'm not sure for the rest of the code : at this place, we do what the runtime requires based on the received runtime name, but we don't modify the runtime name.
The code you add does nothing but fix an unreliable runtime name returned by the function, that's why I think it belongs to the function.

Anyway, no big deal, and it certainly can wait :-)